### PR TITLE
minor copyedits from external contributor (original PR #744)

### PR DESCRIPTION
**Pull request by Bradben**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/concepts/multiple-qubits.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/concepts/multiple-qubits.md) | 73 | 44 | 48 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1b6fb34b-4fb7-4cde-ba16-fe9e57b47317_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/contributing/style-guide.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/contributing/style-guide.md) | 81 | 57 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d30e07db-541b-4e4b-b23e-c3e5a84cf0e7_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/install-guide/pyinstall.md) | 94 | 76 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e57b7325-6392-4aa9-b1a4-8f4612b50903_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/install-guide/qjupyter.md) | 87 | 73 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4eade866-7992-4968-8383-5b589aad35ea_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/install-guide/standalone.md) | 88 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ba34383e-1991-45aa-9232-7182aa15b466_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/machine-learning/design.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/libraries/machine-learning/design.md) | 77 | 45 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eb44860d-b4b4-4538-9127-ca178d23ee7d_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/machines/resources-estimator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/machines/resources-estimator.md) | 87 | 51 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8337a971-0aa5-47e9-8154-651ed334139e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstart.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/quickstart.md) | 86 | 62 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/267a5f73-b864-4a53-9770-13421a1fbd60_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/quickstarts/qrng.md) | 88 | 75 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3da2aef3-e0a6-4714-8dd7-26a1a972c823_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/language/expressions.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/user-guide/language/expressions.md) | 85 | 52 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/376d6515-9f66-4be5-b321-3c25bf474a25_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/language/types.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/user-guide/language/types.md) | 81 | 52 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a6c3de8f-62c0-46cd-992a-783954e5cc24_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/using-qsharp/operations-functions.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/user-guide/using-qsharp/operations-functions.md) | 81 | 57 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/78923c9a-21be-4a06-9d5a-70b5725d9a92_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/using-qsharp/testing-debugging.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/9832883d05353d3da77fce31616435b3de449318/articles/user-guide/using-qsharp/testing-debugging.md) | 81 | 56 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b625152e-400a-4370-aac8-b61c78048b73_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/libraries/machine-learning/design.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Suppose we have a dataset with 784 features per instance, e.g. images of 28×28
 
  ![Local rotations layer](~/media/local_rotations_layer.PNG)
 
-We recommend you to explore the [API referenece of the Quantum Machine Learning library](xref:microsoft.quantum.machinelearning) to discover all the tools available to streamline the circuit design.
+We recommend you  explore the [API reference of the Quantum Machine Learning library](xref:microsoft.quantum.machinelearning) to discover all the tools available to streamline the circuit design.
```
Comment (KittyYeungQ) [inline articles/libraries/machine-learning/design.md:52]:
```suggestion
We recommend you explore the [API reference of the Quantum Machine Learning library](xref:microsoft.quantum.machinelearning) to discover all the tools available to streamline the circuit design.
```

**Comment by KittyYeungQ** [comment]
Thank you @Bradben for addressing this. Will you be editing the api files to update the .yml files in #744 ?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/concepts/multiple-qubits.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/concepts/multiple-qubits.md) | 73 | 44 | 48 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6fb50b53-ead9-433a-b737-2ae299ed44da_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/contributing/style-guide.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/contributing/style-guide.md) | 81 | 57 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/46a86b97-cce3-42ac-af7d-7af7fec8f6fa_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/install-guide/pyinstall.md) | 94 | 76 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/59532a6c-a360-40a5-8fe3-ac1105df9f6b_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/install-guide/qjupyter.md) | 87 | 73 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f11b6b44-4e5f-4564-9452-e5f9d345e9aa_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/standalone.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/install-guide/standalone.md) | 88 | 61 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a90e57e7-3956-4b55-8a9e-413c3fa15242_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/libraries/machine-learning/design.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/libraries/machine-learning/design.md) | 77 | 45 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e7c9f4c1-f2e2-4d28-bc2b-0069252feb52_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/machines/resources-estimator.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/machines/resources-estimator.md) | 87 | 51 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/684789de-b431-4d85-8b63-9549c8d08b0e_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstart.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/quickstart.md) | 86 | 62 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/025f637d-bd36-45bf-b580-c995bf9ac8d8_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/quickstarts/qrng.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/quickstarts/qrng.md) | 88 | 75 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e2264904-b918-40be-82a8-22e1ab5a323f_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/language/expressions.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/user-guide/language/expressions.md) | 85 | 52 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6d94ea73-2908-473c-bde0-5c67937858d3_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/language/types.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/user-guide/language/types.md) | 81 | 52 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e779e612-6529-4427-ae68-e4badd15b65c_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/using-qsharp/operations-functions.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/user-guide/using-qsharp/operations-functions.md) | 81 | 57 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a2f4a123-6c1a-4522-92a6-bec9b9db1dc3_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/user-guide/using-qsharp/testing-debugging.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/80b5ee1834f58a0d279e59e45b699a1fbf3b3ae3/articles/user-guide/using-qsharp/testing-debugging.md) | 81 | 56 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3906234d-63af-42eb-a6c6-0bbf2586257d_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by Bradben** [comment]
Yes, they are PRs in the qsharp-runtime and quantumLibraries repos.

**Pull request merged**